File: app/models/pbx_group_user.rb
class PbxGroupUser < RestModel
    include ActiveModel::Validations

    belongs_to :pbx_group

    property :id
    property :pbx_group_id
    property :pbx_user_id
    property :priority

    validates :pbx_group_id, presence: true
    validates :pbx_user_id, presence: true

end

File: frontend/app/models/pbx-group-user.js
import DS from 'ember-data'
import pbxGroupUserValidations from '../validations/pbxGroupUser'

export default DS.Model.extend(pbxGroupUserValidations,{

  pbxGroup: DS.belongsTo('pbxGroup'),
  pbxGroupId: DS.attr(),
  pbxUserId: DS.attr(),
  priority: DS.attr()
})
